mteb/overview.py

```diff
@@ -124,13 +124,11 @@ def __repr__(self) -> str:
         return "MTEBTasks" + super().__repr__()
 
     @staticmethod
-    def _extract_property_from_task(task, property):
+    def _extract_property_from_task(task, property: str):
```

Comment:
Bonus: Could add this as well to complete the function.
```suggestion
    def _extract_property_from_task(task: AbsTask, property: str):
```